app/controllers/app_groups_controller.rb

```diff
@@ -49,17 +49,45 @@ def show
     @allow_edit_metadata = policy(@app_group).update?
     @allow_edit_app_group_name = policy(@app_group).update_app_group_name?
     @allow_delete_helm_infrastructure = policy(@app_group.helm_infrastructure).delete?
+    @allow_manage_labels = policy(@app_group).update_labels?
+    @required_labels = Figaro.env.DEFAULT_REQUIRED_LABELS.split(',', -1)
```

Comment:
do we use uppercase to call the figaro.env or lowercase? is it possible to call it with uppercase one because I saw the rest use lowercase.

Reply:
was testing using all uppercase mas, it was fine I think